service/relayer/src/app/args.rs

```diff
@@ -0,0 +1,73 @@
+use clap::{Args, Parser};
+use near_crypto::SecretKey;
+use near_sdk::{AccountId, NearToken};
+
+#[derive(Parser, Debug, Clone)]
+pub struct Configuration {
+    /// Run the relayer on this port.
+    #[arg(short, long, env = "PORT", default_value_t = 3000)]
+    pub port: u16,
+    /// Postgres database connection URL.
+    #[arg(long, env = "DATABASE_URL")]
+    pub database_url: String,
+    /// NEAR RPC connection URL.
+    #[arg(long, env = "RPC_URL", default_value = "https://rpc.testnet.near.org")]
+    pub rpc_url: String,
+    #[clap(flatten)]
+    pub monitor: Monitor,
+    /// Account ID of the NEAR account that the relayer controls.
+    #[arg(short, long, env = "ACCOUNT_ID")]
+    pub account_id: AccountId,
+    /// Comma-separated list of private keys to use to sign transactions for the account that the relayer controls.
+    #[arg(short = 'k', long, env = "SECRET_KEY", value_delimiter = ',')]
+    pub secret_key: Vec<SecretKey>,
+    /// Comma-separated list of allowed methods.
+    #[arg(long, env = "ALLOWED_METHODS", default_values_t = default_allowed_methods(), value_delimiter = ',')]
+    pub allowed_methods: Vec<String>,
+    /// Starting allowance in yoctoNEAR.
+    #[arg(long, env = "STARTING_ALLOWANCE_YOCTO", default_value = "0.25 NEAR")]
+    pub starting_allowance_yocto: NearToken,
+    /// Refresh the cached gas price after X seconds.
+    #[arg(long, env = "CACHE_GAS_PRICE_SECS", default_value = "600")]
```

Comment:
I prefer using `default_value_t` for types that support it, since it gives you more compile time checks